### DISCUSSION TITLE
Login not working after following Quick Start Guide

### DISCUSSION THREAD

**Discussion by jmosen** [HCL-TECH-SOFTWARE/connections-automation#86] [Q&A] [answered]
Hi,

First of all I want to say a big thank you, the automation with Ansible is a great thing, especially in combination with Kubernetes and the Component Pack. For setting up test environments we have wanted something like this for a long time.

I managed to work through the quick start guide, which is documented very well. The only problems were due to firewall and network configuration, but I think that is outside the scope of the quick start guides. 

After successfull execution of _playbooks/setup-connections-complete.yml_ and _playbooks/setup-component-pack-complete-single-node.yml_ I have CNX7 + Component Pack up and running. I can access the login page at [https://web.internal.example.com/](https://web.internal.example.com/) which redirects to [https://web.internal.example.org/homepage/login/](https://web.internal.example.org/homepage/login/). Logging in with fakeuser1 and password redirects to [https://web.internal.example.org/homepage/login/](https://web.internal.example.org/homepage/login/) again without any error message. Logging in at [https://web.internal.example.org/social/views/login.html](https://web.internal.example.org/social/views/login.html) results in _{"error":{"statusCode":404,"name":"Error","message":"Cannot POST /social/auth/ibm-connections/login"}}_. 

How do I proceed to find out what's wrong with the login?

Kind Regards
Julian

**Comment by dejo1307** [comment, marked as answer]
Hi Julian,

Welcome to the team :)

Let me 'use you' to test two things:
- What is the browser you are trying to do it in? Can you try in Firefox in a private window, and see if you are still seeing same issue?
- Can you try to run one more playbook - playbooks/hcl/connections-post-install.yml against your Connections inventory file; that one has the fixes which are fixing exactly what you are seeing on Chrome and some other browsers. In case that one fails (that playbook) I know already what is the issue and I know even where it is fixed; just ping me here, and I will push that specific fix for you to test (if you have that problem). Otherwise, that fix is on the list to be pushed with bunch of other stuff probably already this Friday.

Thanks a lot,
Dejan

**Reply by jmosen** [reply]
Wow, thanks for the tip with Firefox. The login works there without problems. After running playbooks/hcl/connections-post-install.yml the login with Chrome works, too.

Now I'm ready to check if everything is running.

Thanks!
